src/test/java/calculator/model/NumberStringConverterTest.java

```diff
@@ -0,0 +1,44 @@
+package calculator.model;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.assertSimpleTest;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import calculator.Application;
+import calculator.view.Message;
+import camp.nextstep.edu.missionutils.test.NsTest;
+import org.junit.jupiter.api.Test;
+
+public class NumberStringConverterTest extends NsTest {
+
+    @Test
+    void 숫자_문자열_배열에_숫자가_아닌_값이_포함되어_있으면_예외발생() {
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException("1:a:2"))
+                        .isInstanceOf(IllegalArgumentException.class)
+                        .hasMessage(Message.INCLUDE_NOT_NUMBER.getMessage() + "a")
+        );
+    }
```

Comment:
NumberStringConverter를 테스트 하신다면, 굳이 NsTest를 이용하실 필요는 없으실 것 같아요~

```suggestion
    @Test
    void 숫자_문자열_배열에_숫자가_아닌_값이_포함되어_있으면_예외발생() {
        NumberStringConverter converter = new NumberStringConverterImpl();
        
        assertThatThrownBy(() -> converter.convert("1:a:2"))
                .isInstanceOf(IllegalArgumentException.class);
    }
```

Reply:
`NsTest`를 사용한 이유는 라이브러리 요구사항에
> 사용자가 입력하는 값은 camp.nextstep.edu.missionutils.Console의 readLine()을 활용한다.
> build.gradle파일은 변경할 수 없으며,제공된 라이브러리 이외의 외부 라이브러리는 사용하지 않는다.

라는 사항이 있었고, 테스트 코드를 처음 작성해봤기 때문에 우테코에서 제공하는 라이브러리를 최대한 활용해서 최대한 요구사항에서 벗어나지 않으려고 했던 시도였습니다.
2주차부터는 추천하는 방법으로도 한번 구현하도록 하겠습니다! 알려주셔서 감사합니다!!!

Reply:
"라이브러리를 최대한 활용"에 방점을 두셨군요 👍